src/refiners/foundationals/latent_diffusion/stable_diffusion_1/model.py

```diff
@@ -29,6 +29,41 @@ class StableDiffusion_1(LatentDiffusionModel):
         unet: The U-Net model.
         clip_text_encoder: The text encoder.
         lda: The image autoencoder.
+
+    Example:
+    ```py
+    import torch
+
+    from refiners.fluxion.utils import manual_seed, no_grad
+    from refiners.foundationals.latent_diffusion.stable_diffusion_1 import StableDiffusion_1
+
+    # Load SD
+    sd15 = StableDiffusion_1(device="cuda", dtype=torch.float16)
+
+    sd15.clip_text_encoder.load_from_safetensors("realistic_vision.v5_1.sd1_5.text_encoder.safetensors")
+    sd15.unet.load_from_safetensors("realistic_vision.v5_1.sd1_5.unet.safetensors")
+    sd15.lda.load_from_safetensors("realistic_vision.v5_1.sd1_5.autoencoder.safetensors")
+
+    # Hyperparameters
+    prompt = "RAW photo, face portrait photo of beautiful 26 y.o woman, cute face, wearing black dress, happy face, hard shadows, cinematic shot, dramatic lighting"
+    seed = 42
+
+    sd15.set_inference_steps(50)
+
+    with no_grad():  # Disable gradient calculation for memory-efficient inference
+        clip_text_embedding = sd15.compute_clip_text_embedding(text=prompt + ", best quality, high quality", negative_text="monochrome, lowres, bad anatomy, worst quality, low quality")
+        manual_seed(seed)
+
+        # SDXL typically generates 1024x1024, here we use a higher resolution.
```

Comment:
```suggestion
```

Reply:
(This comes from [this guide](https://refine.rs/guides/adapting_sdxl/) but it does not concern this example.)